src/data/base/aerodromeLpPools.json

```diff
@@ -1,4 +1,24 @@
 [
+  {
+    "name": "aerodrome-usd+-wsteth",
+    "address": "0xf15B30a0a823f588B523fD794A43939F0B1dC582",
+    "gauge": "0xcC5931E3f7ce8967Dcdc4BC5C7cfd5bF3d7Cf1FE",
+    "decimals": "1e18",
+    "chainId": 8453,
+    "beefyFee": 0.095,
+    "lp0": {
+      "address": "0xB79DD08EA68A908A97220C76d19A6aA9cBDE4376",
+      "oracle": "tokens",
+      "oracleId": "USD+",
```

Comment:
Can you use `baseUSD+` instead? It's the price specific to the Base chain for USD+